### PR TITLE
Excel-importer

### PR DESCRIPTION
**Pull request by du-phan**
https://app.clubhouse.io/dataiku/story/42400/dataset-from-folder-metrics-engine-failed-to-run

### CODE REVIEW

File: excel-importer/python-runnables/macro-excel-importer/runnable.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
+# This file is the actual code for the Python runnable macro-excel-importer
```
Comment (JoachimZ) [inline excel-importer/python-runnables/macro-excel-importer/runnable.py]: lol, not necessary

File: excel-importer/python-runnables/macro-excel-importer/runnable.py [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 import dataiku
 import pandas as pd
 import os
 import openpyxl
 import time
 from dataiku.runnables import Runnable, ResultTable
 
-
 class MyRunnable(Runnable):
+    """The base interface for a Python runnable"""
```
Comment (JoachimZ) [inline excel-importer/python-runnables/macro-excel-importer/runnable.py]: Remove comment

File: excel-importer/python-runnables/macro-excel-importer/runnable.py (hunk 1 above)
Comment (du-phan) [reply excel-importer/python-runnables/macro-excel-importer/runnable.py]: removed!

File: excel-importer/python-runnables/macro-excel-importer/runnable.py (hunk 2 above)
Comment (du-phan) [reply excel-importer/python-runnables/macro-excel-importer/runnable.py]: removed!

**Pull request merged**